core/src/net/sf/openrocket/rocketcomponent/Streamer.java

```diff
@@ -146,8 +156,6 @@ protected void loadFromPreset(ComponentPreset preset) {
 	@Override
 	public double getComponentCD(double mach) {
 		double density = this.getMaterial().getDensity();
-		double cd;
```

Comment:
I don't think you should edit the global cd variable for this, so keep the local cd variable? Correct me if I'm wrong

Reply:
Corrected.